src/main/webapp/js/instructor.es6

```diff
@@ -369,6 +369,25 @@ function attachEventToDeleteStudentLink() {
     });
 }
 
+function remindButtonAjax(urlLink) {
```

Comment:
This name doesn't sound like an action. Is this for consistency/convention reasons? Otherwise rename?